core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/withEditor.ts

```diff
@@ -189,6 +195,16 @@ export function withEditor() {
                               }
                             : null
                     };
+                }),
+                $iframeURL: computed<string>(() => {
```

Comment:
I kwow we want to separate the editorProps in other small states, but what about to  group the iframe variables in 
iframe: {
    url: ..
    opacity: ..
    pointerEvents: ..
    wrapper: ..
}
?

Reply:
The `$iframeURL` should only change when needed. If we add more properties to the `computed`, it will update if any of those properties change.